app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+
+const { getTopics } = require("./controllers/topicsControllers");
+
+const {handleInvalidRoute} = require("./error_handlers/errors")
+
+app.get("/api/topics", getTopics);
+
+app.use("*", handleInvalidRoute);
```

Comment:
Hey Asad - app.use() is used primarily to invoke middleware functions, not to handle endpoints. This will need to be either an app.get( ) or app.all( )

Reply:
Got it. So if I change the code and push it now to this branch, will this PR update or do I have to create a new PR?

Reply:
Just an added thing. If someone were to clone this repo down, they would not be be able to use Insomnia to emulate client requests and you are forcing them to use superTest. If you could amend this and then that would be awesome. There doesn't need to be a new pull request

Reply:
Ok. Thanks